packages/runner-shared/src/event-manager.js

```diff
@@ -527,6 +527,10 @@ export const eventManager = {
       handleBeforeScreenshot(config, callback)
     })
 
+    Cypress.multiDomainCommunicator.on('url:changed', (url) => {
+      Cypress.emit('url:changed', url)
```

Comment:
This will cause the event to be sent (publicly) in the primary domain. Based on our recent discussion about the `viewport:changed` event, I don't think this is the behavior we want. Correct me if I'm wrong, but we only want to emit in the secondary domain when the event occurs in the secondary domain.  

If we emit on `localBus` here instead `Cypress` here, it should update the runner URL UI and not emit the event publicly.

Reply:
updated